internal/pkg/log/debug.go

```diff
@@ -123,7 +123,7 @@ func debugCore(writer *ioutil.AtomicWriter, level zapcore.LevelEnabler) zapcore.
 	}
 	return zapcore.NewCore(
 		newPrefixEncoder(zapcore.NewConsoleEncoder(encoderConfig)),
-		zapcore.Lock(zapcore.AddSync(writer)),
+		zapcore.Lock(writer),
```

Comment:
A este som nasiel jeden problem, ... `zap.AddSync` prida `no operation sync`, ... ale `AtomicWriter` ma `Flush` metodu, co je to iste ako `Sync`, ... takze som doplnil chybajucu metodu `Sync`.